script.js

```diff
@@ -1,4 +1,398 @@
+
+console.log("Dealer Score:");
+console.log(dealerTotal, dealerAceCount);
+
+/*********************************************************************************************************/
+// 10. To give dealer another card when the score is below 17.
+var dealerDrawCard = function () {
+  while (dealerTotal < 17) {
+    let cardImage = document.createElement("img");
+    let card = deck.pop();
+    cardImage.src = "./card/" + card + ".png";
+    getCardValue = cardValue(card);
+    dealerTotal += getCardValue;
+    getAceCardValue = AceCardValue(card);
+    dealerAceCount += getAceCardValue;
+    document.getElementById("dealer-cards").append(cardImage);
+  }
+};
+
+//getDealerDrawCard = dealerDrawCard();
+console.log("Dealer Score:");
+console.log(dealerTotal);
+
+/*********************************************************************************************************/
+// 11. Player to draw card
+var playerDrawCard = function () {
+  for (i = 0; i < 2; i++) {
+    let cardImage = document.createElement("img");
+    let card = deck.pop();
+    cardImage.src = "./card/" + card + ".png";
+    getCardValue = cardValue(card);
+    playerTotal += getCardValue;
+    getAceCardValue = AceCardValue(card);
+    playerAceCount += getAceCardValue;
+    document.getElementById("player-cards").append(cardImage);
+  }
+};
+
+//getPlayerDrawCard = playerDrawCard();
+console.log("Player Score");
+console.log(playerTotal);
+
+/*********************************************************************************************************/
+// 12. Functionality to the Hit button
+var hitButton = function () {
+  if (!toHit) {
+    return;
+  }
+  let cardImage = document.createElement("img");
+  let card = deck.pop();
+  cardImage.src = "./card/" + card + ".png";
+  getCardValue = cardValue(card);
+  playerTotal += getCardValue;
+  getAceCardValue = AceCardValue(card);
+
+  playerAceCount += getAceCardValue;
+  document.getElementById("player-cards").append(cardImage);
+
+  if (getReduceAceValue > 21) {
+    toHit = false;
+    // Eg changing the value of Ace from 11 to 1 if the condition is met
+  }
+};
+
+//clickHitButton = hitButton();
+clickHitButton = document
+  .getElementById("hit-button")
+  .addEventListener("click", hitButton);
+
+/*********************************************************************************************************/
+// 13. To change to Ace count from 10 or 11 to 1 if condition
+
+var reduceAceValue = function (playerTotal, playerAceCount) {
+  if (playerTotal > 21 && playerAceCount == 1) {
+    playerTotal -= 1;
+    playerAceCount += 0;
+  } else if (playerTotal > 21 && playerAceCount >= 2) {
+    playerTotal -= 1;
+    playerAceCount -= 1;
+  } else if (playerTotal < 21 && playerAceCount == 1) {
+    playerTotal -= 0;
+    playerAceCount -= 0;
+  } else if (playerTotal < 21 && playerAceCount >= 2) {
+    playerTotal -= 10;
+    playerAceCount -= 1;
+  }
+  return playerTotal;
+};
+getReduceAceValue = reduceAceValue();
```

Comment:
it's a bit strange to have a random variable being assigned outside of all the functions, maybe find a suitable function to put this line in.